server/src/main/java/org/apache/iotdb/db/metadata/mnode/LeafMNode.java

```diff
@@ -24,14 +24,9 @@
 import org.apache.iotdb.tsfile.read.TimeValuePair;
 import org.apache.iotdb.tsfile.write.schema.MeasurementSchema;
 
-import java.util.Collections;
 import java.util.Map;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-public class LeafMNode extends MNode {
-
-  private static final Logger logger = LoggerFactory.getLogger(LeafMNode.class);
+public class LeafMNode extends InternalMNode {
```

Comment:
Call this MeasurementMNode is better.

Reply:
I agree with that. Otherwise the concept of leaf will be confusing.

Reply:
👍